provider/win_driver_installer_test.py

```diff
@@ -38,6 +40,7 @@
     '"PCI\\VEN_1AF4&DEV_1000" "PCI\\VEN_1AF4&DEV_1041"',
     '"PCI\\VEN_1AF4&DEV_1052"',
     '"ACPI\\VEN_QEMU&DEV_0002"',
+    r'"PCI\VEN_1AF4&DEV_1002" "PCI\VEN_1AF4&DEV_1058"',
```

Comment:
Does it make any difference with the regular expression format? Just curious why this is added in a different way from previous patterns.

Reply:
CI check will fail with the previous patterns, and suggest using this one. Something related to CI  should be updated recently.